Sources/CardPayments/APIRequests/ConfirmPaymentSourceRequest.swift

```diff
@@ -34,8 +34,7 @@ struct ConfirmPaymentSourceRequest: APIRequest {
         confirmPaymentSource.paymentSource = PaymentSource(card: card)
         
         self.orderID = cardRequest.orderID
-        self.accessToken = accessToken
-        
+        self.base64EncodedClientID = Data(clientID.appending(":").utf8).base64EncodedString()
```

Comment:
```suggestion
        self.base64EncodedCredentials = Data(clientID.appending(":").utf8).base64EncodedString()
```
I would rename this variable since it's technically a `base64(username:password)` pair where the password happens to be an empty string.

Reply:
Oh, that's why we need to append ":".

Reply:
addressed in 2813be8fab3d5b7d0f2c0c6787ef7ce7d0c141c1